app/Providers/AppServiceProvider.php

```diff
@@ -23,6 +26,7 @@ public function register()
      */
     public function boot()
     {
-        //
+        View::share('matchdays', Matchday::all());
+        View::share('players', Player::all());
```

Comment:
Do you really need the matchdays and players in all views? If so this can stay, but then you don't need to pass this data in all your Controller-Methods like in PlayerController@index on line 19.
But my guess is that you don't need this data on all views... Therefore you might wanna delete this as it causes unnecessary database-queries which slow down your application